src/client/services/metiers/bff.metier.service.ts

```diff
@@ -0,0 +1,12 @@
+import { HttpClientService } from '~/client/services/httpClient.service';
```

Comment:
pourquoi avoir fait un bff.metier.service ?

Reply:
Dans formationsInitiales j'avais proposé 'export class FormationInitialeService implements FormationInitialeInterface'

Il faudra uniformiser ça soit on garde ma proposition et c'est à faire dans cette pr soit on garde votre proposition et on uniformise en jardinage 😄

Reply:
On peut ajouter ça à notre prochain jardinage, perso je préfère la version sans le "...Interface" au niveau de l'interface parce que c'est du détail d'implem : quand tu manipule le service, tu t'en fous de savoir si c'est une interface ou pas.
Je pense que ce paragraphe de _Clean Code_ résume bien le sentiment : 
> These are sometimes a special case for encodings. For example, say you are building an
ABSTRACT FACTORY for the creation of shapes. This factory will be an interface and will
be implemented by a concrete class. What should you name them? IShapeFactory and
ShapeFactory? I prefer to leave interfaces unadorned. The preceding I, so common in
today’s legacy wads, is a distraction at best and too much information at worst. I don’t
want my users knowing that I’m handing them an interface. I just want them to know that
it’s a ShapeFactory. So if I must encode either the interface or the implementation, I choose
the implementation. Calling it ShapeFactoryImp, or even the hideous CShapeFactory, is preferable to encoding the interface.

À noter que Martins ne préconise **pas** de nommer l'implémentation avec "...Imp", simplement que c'est mieux que de suffixer l'interface. Idéalement tu voudrais plutôt nommer l'implémentation par ce qui la différencie d'une autre implémentation (dans notre cas cette implémentation est spécifique au BFF, si une autre implémentation utilisait une autre source de donnée, ça serait une implémentation différente)

Reply:
effectivement on peut garder l'interface intacte mais j'avoue que bff.metier.service ça m'a paru pas très beau

A rediscuter en jardinage ! :) (je l'ai ajouté sur l'event)